babel.config.js

```diff
@@ -1,9 +1,9 @@
 module.exports = function (api) {
   const isTest = api.env('test');
-  const presets = ['@babel/preset-env', '@babel/preset-react']
+  const presets = ['@babel/preset-env', '@babel/preset-react'];
   const plugins = ['@babel/plugin-transform-runtime'];
```

Comment:
What's the plugin for?

Reply:
@therobinkim It was suggested in the babel docs ([https://babeljs.io/docs/en/babel-plugin-transform-runtime](url)) to run the plugin alongside `@babel/runtime` for async code - I just removed the line and don't see a noticeable difference. I'll remove the plugin in the next PR - thank you!

Reply:
@willresen (FYI that hyperlink didn't actually work for me; I had to copy/paste the text into my URL bar.) `@babel/runtime` seems useful according to the docs (whereas `@babel/plugin-transform-runtime` seems to only affect your dev environment? not sure).. I wouldn't just take it out necessary, but I'd experiment with it to see how your development and production bundle sizes change as the number of files you're putting through Babel increases over time.

Reply:
@therobinkim My bad for the broken link - not sure what happened there. Great suggestion, I'll toggle it occasionally to check the difference.

Reply:
@therobinkim Ah! I remember now why I installed it. Without the transformer, I recieve `ReferenceError: regeneratorRuntime is not defined` when running Jest tests on async code.